storageclass/storageclass.go

```diff
@@ -62,3 +62,12 @@ func (s *Service) Delete(ctx context.Context, name string) error {
 
 	return nil
 }
+
+// DeleteAll deletes all storageClassses.
+func (s *Service) DeleteAll(ctx context.Context) error {
+	if err := s.client.StorageV1().StorageClasses().DeleteCollection(ctx, metav1.DeleteOptions{}, metav1.ListOptions{}); err != nil {
+		return xerrors.Errorf("delete all storageClassses: %w", err)
```

Comment:
typo (the same typo on the `DeleteAll` comment.)

```suggestion
		return xerrors.Errorf("delete all storageClasses: %w", err)
```